packages/shared/src/lib/featureManagement.ts

```diff
@@ -122,8 +128,30 @@ export class Features {
 
   static readonly MyFeedOnboardingVersion = new Features(
     'my_feed_onboarding_version',
-    'control',
-    ['control', 'v1', 'v2', 'v3'],
+    MyFeedOnboardingVersionEnum.Control,
+    [
+      MyFeedOnboardingVersionEnum.Control,
+      MyFeedOnboardingVersionEnum.V1,
+      MyFeedOnboardingVersionEnum.V2,
+      MyFeedOnboardingVersionEnum.V3,
+    ],
+  );
+
+  static readonly OnboardingSteps = new Features(
+    'onboarding_steps',
+    'topics/layout/theme',
+  );
+
+  static readonly UserOnboardingVersion = new Features(
+    'onboarding_version',
+    OnboardingVersion.V1,
+    [OnboardingVersion.V1, OnboardingVersion.V2],
+  );
```

Comment:
What makes this difference from the myfeed onboarding version?

Reply:
My feed onboarding version is the old one, not sure if it is still going on. I believe some of its values will determine whether to show the skip button or not.